usr/src/uts/common/fs/zfs/spa.c

```diff
@@ -1254,6 +1270,14 @@ spa_unload(spa_t *spa)
 	}
 
 	/*
+	 * Stop autotrim tasks.
```

Comment:
Damn, I thought I had already done this. Moved it above the vdev_metaslab_fini block.